packages/gestalt/src/Text.js

```diff
@@ -21,16 +21,74 @@ type Overflow = 'normal' | 'breakWord' | 'noWrap';
 type Size = 'sm' | 'md' | 'lg';
 
 type Props = {|
-  align?: Align,
+  /**
+   * `"start"` and `"end"` should be used for regular alignment since they flip with locale direction. `"forceLeft"` and `"forceRight"` should only be used in special cases where locale direction should be ignored, such as tabular or numeric text.
+   *
+   * Default: 'start'
+   * Link: https://gestalt.pinterest.systems/text#align
+   */
+  align?: 'start' | 'end' | 'center' | 'justify' | 'forceLeft' | 'forceRight',
```

Comment:
@dangerismycat as soon as you've created the babel plugin to be able to use imported types, we can change this back.